templates/opps-list-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template is used to display the list of flexible volunteer opportunities
+ * for the widget.
+ *
+ * To adjust this template copy it into your current theme within a folder called "wivm".
+ */
+
+// Global variables that are set in widget/class-widget.php via the widget settings in admin 
```

Comment:
@jg314 
**Pull request updated**:
I condensed the global variables into one global array variable ($wivm_opp_options) and updated the widget template to only use this one global variable. Great suggestion.